lib/stripe/util.rb

```diff
@@ -91,47 +91,50 @@ def self.convert_to_stripe_object(data, opts = {})
 
     def self.log_error(message, data = {})
       if !Stripe.logger.nil? ||
-          !Stripe.log_level.nil? && Stripe.log_level <= Stripe::LEVEL_ERROR
+         !Stripe.log_level.nil? && Stripe.log_level <= Stripe::LEVEL_ERROR
         log_internal(message, data, color: :cyan,
-          level: Stripe::LEVEL_ERROR, logger: Stripe.logger, out: $stderr)
+                                    level: Stripe::LEVEL_ERROR, logger: Stripe.logger, out: $stderr)
       end
     end
 
     def self.log_info(message, data = {})
       if !Stripe.logger.nil? ||
-          !Stripe.log_level.nil? && Stripe.log_level <= Stripe::LEVEL_INFO
+         !Stripe.log_level.nil? && Stripe.log_level <= Stripe::LEVEL_INFO
         log_internal(message, data, color: :cyan,
-          level: Stripe::LEVEL_INFO, logger: Stripe.logger, out: $stdout)
+                                    level: Stripe::LEVEL_INFO, logger: Stripe.logger, out: $stdout)
       end
     end
 
     def self.log_debug(message, data = {})
       if !Stripe.logger.nil? ||
-          !Stripe.log_level.nil? && Stripe.log_level <= Stripe::LEVEL_DEBUG
+         !Stripe.log_level.nil? && Stripe.log_level <= Stripe::LEVEL_DEBUG
         log_internal(message, data, color: :blue,
-          level: Stripe::LEVEL_DEBUG, logger: Stripe.logger, out: $stdout)
+                                    level: Stripe::LEVEL_DEBUG, logger: Stripe.logger, out: $stdout)
       end
     end
 
     def self.file_readable(file)
       # This is nominally equivalent to File.readable?, but that can
       # report incorrect results on some more oddball filesystems
       # (such as AFS)
-      begin
-        File.open(file) { |f| }
-      rescue
-        false
-      else
-        true
-      end
+
+      File.open(file) { |f| }
+    rescue
+      false
+    else
+      true
     end
 
     def self.symbolize_names(object)
       case object
       when Hash
         new_hash = {}
         object.each do |key, value|
-          key = (key.to_sym rescue key) || key
+          key = (begin
+                   key.to_sym
+                 rescue
+                   key
+                 end) || key
```

Comment:
I don't like what autocorrect did here either -- the one-liner was pretty straightforward and legible.

Reply:
Yeah this is crazy haha. Curious to see what it does after we remove the "aligned indent" rule or whatever it is.

Reply:
I can't find the rule that caused this, but maybe using `respond_to?` would be better practice anyway?

Reply:
> I can't find the rule that caused this, but maybe using respond_to? would be better practice anyway?

Yeah, much better. Also, using an inline `rescue` like that was a little sketchy to start. Should be fine.